base/codeintel-db/codeintel-db.Deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Postgres database for various data.
+  labels:
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
+  name: codeintel-db
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: codeintel-db
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        deploy: sourcegraph
+        app: codeintel-db
+        group: backend
+    spec:
+      initContainers:
+      - name: correct-data-dir-permissions
+        image: sourcegraph/alpine:3.10@sha256:4d05cd5669726fc38823e92320659a6d1ef7879e62268adec5df658a0bacf65c
+        command: ["sh", "-c", "if [ -d /data/pgdata-11 ]; then chmod 750 /data/pgdata-11; fi"]
+        volumeMounts:
+        - mountPath: /data
+          name: disk
+        securityContext:
+          runAsUser: 0
+      containers:
+      - env:
+        image: index.docker.io/sourcegraph/codeintel-db@sha256:63090799b34b3115a387d96fe2227a37999d432b774a1d9b7966b8c5d81b56ad
+        terminationMessagePolicy: FallbackToLogsOnError
+        readinessProbe:
+          exec:
+            command:
+              - /ready.sh
+        livenessProbe:
+          initialDelaySeconds: 15
+          exec:
+            command:
+              - /liveness.sh
+        name: pgsql
+        ports:
+        - containerPort: 5432
+          name: pgsql
+        resources:
+          limits:
+            cpu: "4"
+            memory: 2Gi
+          requests:
+            cpu: "4"
+            memory: 2Gi
+        volumeMounts:
+        - mountPath: /data
+          name: disk
+        - mountPath: /conf
+          name: pgsql-conf
+      - env:
+        - name: DATA_SOURCE_NAME
+          value: postgres://sg:@localhost:5432/?sslmode=disable
+        image: wrouesnel/postgres_exporter:v0.7.0@sha256:785c919627c06f540d515aac88b7966f352403f73e931e70dc2cbf783146a98b
+        terminationMessagePolicy: FallbackToLogsOnError
+        name: pgsql-exporter
```

Comment:
- [ ] We should scrape the metrics for this new container; @sourcegraph/distribution please advise

Reply:
This should be automatically discovered as its defined in `codeintel-db.Service.yaml`